moderatorguide.md

```diff
@@ -45,20 +47,18 @@ In general, mediating questions works like this:
 * In the event of a prolonged discussion, you may get the talk back on track by suggesting that the conversation continue separately after the seminar.
 
 
-### Deal with technical issues
-
-Just as in a non-virtual seminar, technical issues may crop up. Here, they can take the form of degraded/lost internet connection, computer failure, etc. Where possible, the moderator should do their best to resolve or manage technical issues that disrupt a seminar.
+### Technical issues
 
 * Speaker-side issues: where possible, try to have a second line of communication with the speakers in case an issue comes up on their end.
 * Audience-side issues: attendees are instructed to send chat messages directly to the moderator if technical issues arise on their end, and the moderator must make a judgement call as to whether some kind of action should be taken.
 
 Moderators should report on any such issues (and fixes) to the organizers or the VSF team, and also encourage such feedback from all participants.
 
 
-### Deal with misconduct
+### Misconduct
 
-In the event of misconduct (defined by the code of conduct) from any participant moderators need to take the following actions:
+In the event of misconduct from any participant moderators need to take the following actions:
 
 * Warn involved participant(s) of their behaviour and the possible consequences.
 * Remove repeat offenders from the session.
-* Overlook chat interactions where the same rules apply.
\ No newline at end of file
+* Overlook chat interactions where the same rules apply.
```

Comment:
This doesn't seem to belong to the list: it's not a response to violation of conduct. (as this stands, we're listing actions to take *in case of code of conduct breach*)

Reply:
I agree this bullet item is unnecessary. I've made an edit suggestion above that explicitly states the venues in which misconduct can occur, including the text chat.

Reply:
```suggestion

```